systems/init/msgClient/pb/msgClient.proto

```diff
@@ -0,0 +1,88 @@
+syntax = "proto3";
+
+option go_package = "pb/gen";
+
+package ukama.msgClient.v1;
+
+import "validator.proto";
+import "google/protobuf/any.proto";
+
+/*
+MsgClient system is responsibe for inter systems comunications, Suystems can emmit or generate events using MsgClientService. System expose following rpc's:
+    - Register Service 
+    - StartMsgBusHandler
+    - StopMsgBusHandler
+    - PublishMsg
+*/
+
+service MsgClientService {
+    /// Use this rpc to register system to MsgClient
+    rpc RegisterService(RegisterServiceReq) returns (RegisterServiceResp);
+    /// Call this rpc to  StartMsgBus after registration
+    rpc StartMsgBusHandler(StartMsgBusHandlerReq) returns (StartMsgBusHandlerResp);
+    /// Call this rpc to  StopMsgBus
+    rpc StopMsgBusHandler(StopMsgBusHandlerReq) returns (StopMsgBusHandlerResp);
+    /// Unregister service from MsgClient
+    rpc UnregisterService(UnregisterServiceReq) returns (UnregisterServiceResp);
+    /// Call this rpc to publisg events
+    rpc PublishMsg(PublishMsgRequest) returns (PublishMsgResponse);
+
+}
+
+/// Registration status enums
+enum REGISTRAION_STATUS {
+    REGISTERED = 0; /// System registered status
+    ALLREADY_REGISTERED = 1; /// System already registered status
```

Comment:
I guess you meant "ALREADY_REGISTERED". Also, isn't an "ALREADY_REGISTERED" status a "REGISTERED" status?

Reply:
yeah. Idea was to get ALREADY_REGISTERED when service was previously succesfully registered, But its not required anymore. So removed.